lib/matplotlib/tests/test_axes.py

```diff
@@ -4931,3 +4931,15 @@ def test_bar_single_height():
     ax.bar(range(4), 1)
     # Check that a horizontal chart with one width works
     ax.bar(0, 1, bottom=range(4), width=1, orientation='horizontal')
+
+
+def test_invalid_axis_limits():
```

Comment:
One last thing, could a `@cleanup` decorator go above this line. Otherwise looks good!